uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/Habit.kt

```diff
@@ -59,9 +59,10 @@ data class Habit(
         val today = DateUtils.getTodayWithOffset()
         val value = computedEntries.get(today).value
         return if (isNumerical) {
+            val targetValuePerDay = (targetValue / frequency.denominator)
             when (targetType) {
-                NumericalHabitType.AT_LEAST -> value / 1000.0 >= targetValue
-                NumericalHabitType.AT_MOST -> value / 1000.0 <= targetValue
+                NumericalHabitType.AT_LEAST -> value / 1000.0 >= targetValuePerDay
+                NumericalHabitType.AT_MOST -> value / 1000.0 <= targetValuePerDay
```

Comment:
Could you please revert these changes for now? It's still unclear to me how should we handle `isCompletedToday` for numeric habits. I unfortunately don't think hiding by `targetValuePerDay` works. I think we will eventually need to do something similar to the score computation, where we actually sum over previous days. In any case, this is unrelated to this PR.